minimesos/src/main/java/com/containersol/minimesos/docker/DockerClientFactory.java

```diff
@@ -12,6 +12,8 @@
 
     private static DockerClient dockerClient;
 
+    private final static String bad_example = "bad example";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "bad_example" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS00115)